docs/json/sonarr/cf/dubs-only.json

```
@@ -25,12 +25,12 @@
       }
     },
     {
-      "name": "KaiDubs",
+      "name": "KaiDubs (Not Dual Audio)",
       "implementation": "ReleaseTitleSpecification",
       "negate": false,
       "required": false,
       "fields": {
-        "value": "\\b(KaiDubs)\\b"
+        "value": "^(?!.*(dual[ ._-]?audio|[\\[(]dual[\\]]|(JA|ZH)\\+EN|EN\\+(JA|ZH))).*\\b(KaiDubs)\\b"
```

```suggestion
        "value": "^(?!.*(dual[ ._-]?audio|[([]dual[])]|(JA|ZH|KO)\\+EN|EN\\+(JA|ZH|KO))).*\\b(KaiDubs)\\b"
```
I think we should actually use the new `Dual Audio` regex

---

Should this be done for the existing KS (Not Dual Audio) portion of the CF as well?

---

would probably make sense yes

---

Updated KS (Not Dual Audio) based on your suggestion